azurecontainerapps/src/TelemetryHelper.ts

```diff
@@ -0,0 +1,67 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import { Utility } from './Utility';
+
+const ORYX_CLI_IMAGE: string = "mcr.microsoft.com/oryx/cli:debian-buster-20230207.2";
+
+export class TelemetryHelper {
+    readonly disableTelemetry: boolean = false;
+
+    private scenario: string;
+    private result: string;
+    private taskStartMilliseconds: number;
+
+    constructor(disableTelemetry: boolean) {
+        this.disableTelemetry = disableTelemetry;
+        this.scenario = "N/A";

```

Comment:
Do we have to always use a string value? If not, please consider using null/undefined as the `non assigned` equivalent

Reply:
I'll leave this property uninitialized and use `isNullOrEmpty` to check if we should log it in telemetry (same with `result` just to be safe if we decide to update it in the future)